module4/model/src/main/java/com/epam/esm/entity/Certificate.java

```diff
@@ -5,6 +5,7 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import lombok.ToString;
 
 import javax.persistence.*;
```

Comment:
is there a lot of imports? check ide configuration

Reply:
my ide has correct configuration, done